extensions/replication/utils/SetupBackbeatClient.js

```diff
@@ -0,0 +1,175 @@
+const http = require('http');
+const errors = require('arsenal').errors;
+const jsutil = require('arsenal').jsutil;
+const VaultClientCache = require('../../../lib/clients/VaultClientCache');
+const BackbeatClient = require('../../../lib/clients/BackbeatClient');
+const BackbeatTask = require('../../../lib/tasks/BackbeatTask');
+const {
+    StaticFileAccountCredentials,
+    ProvisionedServiceAccountCredentials,
+} = require('../../../lib/credentials/AccountCredentials');
+const { attachReqUids } = require('../../../lib/clients/utils');
+const RoleCredentials = require('../../../lib/credentials/RoleCredentials');
+
+class SetupBackbeatClient extends BackbeatTask {
```

Comment:
Since this is a helper class that wraps the low-level backbeat client with its own functions, it should have a name reflecting this, could be called `BackbeatMetadataProxy` instead (for example, suggestions welcome!)